tests/phpunit/tests/option/transient.php

```diff
@@ -87,36 +87,6 @@ function test_transient_add_timeout() {
 		$this->assertFalse( get_transient( $key ) );
 	}
 
-	/**
-	 * If get_option( $transient_timeout ) returns false, don't bother trying to delete the transient.
-	 *
-	 * @ticket 30380
-	 */
-	function test_nonexistent_key_dont_delete_if_false() {
```

Comment:
The need to delete this suggests the code reintroduces the issue in https://core.trac.wordpress.org/ticket/30380. I've added some notes to this pull requests trac ticket, [WP#52345](https://core.trac.wordpress.org/ticket/52345), with an alternative approach that will allow you to retain this test.

Reply:
I've implemented your suggested change.
See the Trac ticket for more information.